layouts/partials/nav_links.html

```diff
@@ -0,0 +1,71 @@
+<nav class="next-previous-links">
+    {{ $sectionParentPages := .CurrentSection.Parent.Pages }}
+    {{ $currentSection := .CurrentSection.Title }}
+
+
+    <sl-divider style="--spacing: 4rem 0rem;">></sl-divider>
+    <div class="nav-links">
+        <div>
+            {{ if .NextInSection }}
+                {{ with .NextInSection }}
+                    <a href="{{ .Permalink }}" class="nav-links-previous"> Previous: </a>
+                    <div>{{ .Title }}</div>
+                {{ end }}
+
+            {{ else }}
+                {{ range $sectionParentPages }}
+                    {{ if eq .Title $currentSection }}
+                        {{ with $sectionParentPages.Next . }}
```

Comment:
I don't understand how `.Next` returns the previous page

Reply:
[.Next points up](https://gohugo.io/variables/pages/) to the next page relative to the page sent in as the argument. the list of parent sections is fetched in the order as they appear in the side bar, so .Next in this context will return the section above, which is the previous section from our point of view

Reply:
No, I *think*, it's due to hugo's original blog origins where _reverse chronological_ is the default sort order: https://discourse.gohugo.io/t/next-and-previous-pages-reversed-meaning/25598/2

Happy to be proven wrong. Regardless, comment in code